evalml/automl/automl_search.py

```diff
@@ -416,7 +418,7 @@ def search(self, X, y, data_checks="auto", feature_types=None, show_iteration_pl
             logger.info("Generating pipelines to search over...")
             allowed_estimators = get_estimators(self.problem_type, self.allowed_model_families)
             logger.debug(f"allowed_estimators set to {[estimator.name for estimator in allowed_estimators]}")
-            self.allowed_pipelines = [make_pipeline(X, y, estimator, self.problem_type) for estimator in allowed_estimators]
+            self.allowed_pipelines = [make_pipeline(X, y, estimator, self.problem_type, text_columns=text_columns) for estimator in allowed_estimators]
```

Comment:
Looks good. I will make a note for the woodwork integration that we should have `make_pipeline` work with datatables instead of pandas types, so that we don't need to pass `text_columns` in here!